qubes/ext/gui.py

```diff
@@ -33,3 +33,13 @@ def send_gui_mode(vm):
             input=('SEAMLESS'
             if vm.features.get('gui-seamless', False)
             else 'FULLSCREEN'))
+
+    @qubes.ext.handler('domain-qdb-create')
+    def on_domain_qdb_create(self, vm, event):
+        # pylint: disable=unused-argument,no-self-use
+        for feature in ('gui-videoram-overhead', 'gui-videoram-min'):
+            try:
+                vm.untrusted_qdb.write('/qubes-{}'.format(feature),
+                    vm.features.check_with_template(feature))
```

Comment:
It would be good to have global default for this, not only per-template (you want to use all your VMs with external 4K display, right?). With current API, that could be a feature on `dom0`, but that isn't the most elegant solution. Maybe we need also features on `Qubes()` itself?

Reply:
Yes, at least `check-updates` (default was promoted to `Qubes()`'s property for this reason), but I think that problem will happen from time to time.

Reply:
This looks like change for 5.0. For example, this throws events `domain-feature-{set,delete}`, which should be changed if features are to be reused on app object. Admin API change should be easy.

Reply:
I see. So maybe use features on dom0 for now?

Reply:
There is no "for now". If I do it like that, it will remain. :)

Reply:
Here is your global knob.

Please carefully review the commit about features.